### PR TITLE
TW 42 Update /collaborating-in-postman/roles-and-permissions/

### PR DESCRIPTION
**Pull request by HeatherStenson**
- Adds information about role-based permissions for monitors and mock servers
- General rewriting, rearranging, and editing

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 1]
```diff
@@ -27,47 +27,41 @@ contextual_links:
...
 * **Admin**: Manages team members and team settings.
-* **Billing**: Manages team plan and payments.
+* **Billing**: Manages team plan and payments. Billing roles can be granted by a Super Admin or by a fellow team member with a Billing role. If this is not possible, contact our [Support Center](https://support.postman.com/hc/en-us).
 * **Developer**: Has access to all team resources and workspaces.
-* **Community Manager**: Manages public visibility of workspaces and team profile (Professional and Enterprise plans only).
+* **Community Manager**: Manages the public visibility of workspaces and team profile ([Professional and Enterprise plans only](https://www.postman.com/pricing)).
 
-Each user must have at least one role attached to them, and can hold multiple roles simultaneously.
+> If you are on a [Postman Enterprise plan](https://www.postman.com/pricing), you can also assign roles at the [group level](/docs/administration/managing-your-team/user-groups/) as well as for individual team members.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:64]:
```suggestion
> If you are on a [Postman Enterprise plan](https://www.postman.com/pricing), you can also assign roles at the [group level](/docs/administration/managing-your-team/user-groups/).
```

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 2]
```diff
@@ -93,9 +87,15 @@ Team roles provide high-level access control:
 
 &ast;&ast; Enterprise and Professional plans only
 
-### Workspace roles
+> **Postman Support Users**. Members with a Developer role consume a paid slot on the team. Team members who have only Admin or Billing roles become Support Users and don’t consume paid slots. Your team can have two Support Users.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:90]: I'm not sure if "Support Users" should be capitalized, maybe not unless we can locate this term capitalized elsewhere in Postman app or HC? The capitalization to me makes it come across as an actual role one could assign.

```suggestion
> **Postman Support Users**. Members with a Developer or Super Admin role consume a paid slot on your team. Team members who have only Admin or Billing roles become Support Users and don’t consume paid slots. Your team can have two Support Users.
```

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 3]
```diff
@@ -145,47 +169,42 @@ The following roles control access at an API level:
...
-| Merge forks on collections  |    &#x2714;     |    |
-| Publish collection documentation and add to API Network  |   &#x2714;  |   |
-| Share collections to a different workspace  |  &#x2714;  | &#x2714;   |
-| Tag and restore collection versions   |  &#x2714;   |   |
-| Add, edit, and delete mock servers  |    &#x2714;      |   |
-| Add, edit, and delete monitors |    &#x2714;   |     |
+You can [assign](/docs/administration/managing-your-team/managing-your-team/) two role types for Postman mock servers: **Editor** and **Viewer**.
 
-## Managing roles and permissions
+* **Editor**: Can edit mock servers directly
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:176]:
```suggestion
* **Editor**: Can edit and manage mock servers
```

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 4]
```diff
@@ -145,47 +169,42 @@ The following roles control access at an API level:
...
+View monitor based integrations | &#x2714; | &#x2714;
+Edit and delete monitor | &#x2714; |
+Run, pause, and resume monitor | &#x2714; |
+Move monitor | &#x2714; |
+Update monitor roles | &#x2714; |
 
 ## Next steps
 
-Learn more about [Working with your team](/docs/collaborating-in-postman/collaboration-intro/).
+Learn more about [Managing your team](/docs/administration/managing-your-team/managing-your-team/), including managing roles and inviting people to join your team.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:210]:
```suggestion
Learn more about [Managing your team](/docs/administration/managing-your-team/managing-your-team/), including managing team roles and inviting collaborators to join your team.
```

**Review by claireriley — APPROVED** [review]
The reorganization and additional sections look great @HeatherStenson ! 🥳  Couple minor wording suggestions and one term to look in to prior to publishing

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md (hunk 2 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/roles-and-permissions.md]: I agree that capitalizing Support Users makes it seem like an assignable role. It _is_ capitalized in the Postman app:
Screen Shot 2022-01-31 at 10 28 18 AM

But overall I think it makes mores sense to lower case this term in the LC (and possibly open a ticket to change this in the Postman UI).

**Pull request merged**